score/mw/com/impl/configuration/config_parser.cpp

```diff
@@ -225,7 +278,8 @@ auto ParseShmSizeCalcMode(const score::json::Any& json) -> score::cpp::optional<
         else
         {
             score::mw::log::LogError("lola")
-                << "Unknown value " << shm_size_calc_mode_value << " in key " << kShmSizeCalcModeKey;
+                << "Configuration error: Invalid shm-size-calc-mode value '" << shm_size_calc_mode_value
+                << "'. Valid values are: 'SIMULATION'. Please update your configuration file.";
             /* Terminate call tolerated.See Assumptions of Use in mw/com/design/README.md*/
             std::terminate();
```

Comment:
Would be nice if you fix this one as well:

```suggestion
            SCORE_LANGUAGE_FUTURECPP_ASSERT_PRD(false);
```

Reply:
Changing this, but it looks like I didn't introduce that one (just fyi).